basiclti/tsugi-util/src/test/org/tsugi/basiclti/BasicLTIUtilTest.java

```diff
@@ -557,4 +557,21 @@ public void testMergeCSV() {
 		retval = BasicLTIUtil.mergeCSV("1,2", "2,1,3,4", null);
 		assertEquals(retval, "1,2,3,4");
 	}
+
+	@Test
+	public void testShiftJVMDateToTimeZone() {
+		// tz=sun.util.calendar.ZoneInfo[id="America/New_York",offset=-18000000,...
+		TimeZone tz = TimeZone.getTimeZone("America/New_York");
+		assertEquals(tz.getRawOffset(), -18000000);
```

Comment:
what happens during DST?

Reply:
If things go wonky, then the unit test breaks and we know about it.  And fix the unit test.   If this code breaks it is good to know about that.  Part of what I put in these unit tests was my worry about things shifting around on us in future versions of the JVM, etc.  I worry too - so I want an obvious way to track that to call attention to when the rules change.

Reply:
ok i'm fine with it.... just hurts to see you using all of the old bad practices ;)